internal/daysteps/daysteps.go

```diff
@@ -1,13 +1,59 @@
 package daysteps
 
+import (
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/Yandex-Practicum/tracker/internal/personaldata"
+	"github.com/Yandex-Practicum/tracker/internal/spentenergy"
+)
+
 type DaySteps struct {
 	// TODO: добавить поля
+	Steps        int
+	Duration     time.Duration
+	Personal personaldata.Personal
 }
 
 func (ds *DaySteps) Parse(datastring string) (err error) {
 	// TODO: реализовать функцию
+	value := strings.Split(datastring, ",")
+	if len(value) != 2 {
+		return errors.New("length of slice should be equal to 2")
+	}
+	ds.Steps, err = strconv.Atoi(value[0])
+	if err != nil {
+		return err
+	}
+	if ds.Steps <= 0 {
+		return errors.New("количество шагов должно быть больше 0")
+	}
+
+	ds.Duration, err = time.ParseDuration(value[1])
+	if err != nil {
+		return errors.New("error ocured during the time parsing")
```

Comment:
А тут забыл про ошибку
```suggestion
		return fmt.Errorf("invalid duration: %w", err)
```

Reply:
Спасибо, всё исправил